lib/common/formats.js

```diff
@@ -55,9 +55,20 @@ function formattedVariables(format, properties) {
       break;
   }
 
-  return properties.map(function(prop) {
+  return dictionary.allProperties.map(function(prop) {
       var to_ret_prop = prefix + prop.name + ': ';
-      to_ret_prop += (prop.attributes.category==='asset' ? '"'+prop.value+'"' : prop.value);
+      let value = prop.value;
+
+      if (useVariables && dictionary.usesReference(prop.original.value)) {
+        var ref = dictionary.getReference(prop.original.value).name;
+        if (format === 'css') {
+          value = `var(${prefix}${ref})`
+        } else {
+          value = prefix + ref;
```

Comment:
for consistency with two lines above I would do:
```suggestion
          value = `${prefix}${ref}`;
```